### DISCUSSION TITLE
Change primary navigation intro text

### DISCUSSION THREAD

**Discussion by Chewbathra** [area17/twill#1331] [Q&A] [answered]
Hello,

I am struggling changing primary navigation intro text. 

I created a custom setting page with a primary navigation item but can't change the intro text : 
![image](https://user-images.githubusercontent.com/10773783/146997689-1ef7847b-39f7-4df9-bdb0-cca4476c3b8d.png)

Here is my twill-navigation.php: 
```
'settings' => [
  'title' => 'Paramètres',
  'route' => 'admin.settings',
  'params' => ['section' => 'home', ],
  'primary_navigation' => [
      'home' => [
          'title' => 'Paramètres de la page d\'accueil',
      ],
  ]
]
```

I thing i need to add a parameter to my primary navigation, but don't know which one.

Thanks for your help !

**Comment by pboivin** [comment, marked as answer]
Hi @schupbachl, on regular forms this is normally the title editor (with a pencil icon) and corresponds to the current item's title. For settings pages, this is generated automatically but you can change it by passing the `customTitle` attribute to the layout :

```
@extends('twill::layouts.settings', [
    'customTitle' => 'Hello World'
])

@section('contentFields')
    ...
@stop
```

**Reply by Chewbathra** [reply]
Thanks a lot !
